docs/usage.rst

```diff
@@ -162,6 +162,67 @@ mapping that is checked against the :ref:`schema <schema_dict-rule>` rule:
    ``allow_unknown`` can also be set to a validation schema.
 
 
+.. _requiring-all:
+
+Requiring all
```

Comment:
this looks like copied from the `allow_unknown` section. given that `require_all` is less complex, a sufficient explanation should be doable with less prosa.

Reply:
Have no idea how to reduce text. Exclude some examples?